packages/calcite-components/src/components/input-time-zone/input-time-zone.e2e.ts

```diff
@@ -40,7 +40,7 @@ describe("calcite-input-time-zone", () => {
   });
 
   describe("reflects", () => {
-    reflects("calcite-input-time-zone", [
+    reflects.skip("calcite-input-time-zone", [
```

Comment:
Does this skip the test? It's a custom function. If not, this needs to go on the `describe` above.

Reply:
whooops